packages/evasive-transform/src/generate.js

```diff
@@ -4,6 +4,7 @@
  * @module
  */
 
+// @ts-ignore XXX no types defined
```

Comment:
nit: prefer `@ts-expect-error` over `@ts-ignore`.  the only time I use the latter is when I'm seeing flaky behavior